pages/instructorRequestCourse/instructorRequestCourse.ejs

```diff
@@ -3,6 +3,16 @@
     <head>
         <%- include('../partials/head'); %>
         <link href="/stylesheets/issues.css" rel="stylesheet" />
+        <script>
+          $(function() {
+             $('input[name=cr-role]').change(function() {
+                var role = this.value;
+                $('.question-form button').prop('disabled', role != "instructor");
+                $('.role-comment').hide(); 
+                $('.role-comment-' + role).show();
+            });
+          });
+        </script>
```

Comment:
Nice! This seems like a nice solution for this page. As an informational note, we generally try and minimize the use of client-side JS to improve robustness. The normal pattern we use is to have a dumb HTML form that submits to the server and lets the server show a response page with an error message.